wordpress-mobile/WordPress-Android
# Issue/7464 display site quota

Fixes #7464 

This PR updates the project fluxC version to use one that supports site quota information and updates the UI of SiteSettings to show the same information.

![screenshot_1520723925](https://user-images.githubusercontent.com/651601/37252847-90d87e66-2520-11e8-8454-2d0f2569bb93.png)


To test:
 - Start the app
 - Open a WP.com site
 - Go to the settings section
 - See if media quota space information shows up.
